raiden/network/transport/matrix.py

```diff
@@ -181,7 +181,7 @@ def start(
     ):
         self._running = True
         self._raiden_service = raiden_service
-        config = raiden_service.config['matrix']
+        config = raiden_service.config['transport']['matrix']
```

Comment:
Could you, please, set config to `self.config` and use it here, instead of `raiden_service.config`, as per the other comment, and also the other places where we access` raiden_service.config` directly?